include/boost/url/segments_encoded.hpp

```diff
@@ -77,15 +77,9 @@ class segments_encoded
 
 public:
 #ifdef BOOST_URL_DOCS
-    /** A random-access iterator referencing segments in a url path
-
-        When dereferenced, this iterator returns a
-        proxy which allows conversion to stringlike
-        types, assignments which change the underlying
-        container, and comparisons.
```

Comment:
oh yeah this doc is way out of date...we experimenting with an assignment-capable proxy and the results were terrible.

Reply:
Mmm... I didn't know about that. 

I'll remove that. I thought the intention was to still implement something like that in the future.

Reply:
hell no it was painful to implement and awkward to use, due to limitations in the language